### PR TITLE
add alerts to alerts branch

### PR DESCRIPTION
**Pull request by shivamSspirit**

### CODE REVIEW

**Review by zabihshaik — APPROVED** [review]
Everything looks good to merge.

File: Components/alerts/alert.css [hunk 1]
```diff
@@ -0,0 +1,416 @@
...
+.alerts {
+	display: flex;
+	flex-direction: column;
+}
+
+/* .alerts div{
+    margin-top: 0.5rem;
+} */
+
+/* basic alert */
```
Comment (codewithrishu) [inline Components/alerts/alert.css]: remove unwanted code

File: Components/alerts/alert.css [hunk 2]
```diff
@@ -0,0 +1,416 @@
...
+	font-weight: var(--main-text-weight);
+	color: var(--primary-color);
+}
+
+.basic-alerts div {
+	margin-top: 1rem;
+}
+
+.Basic-Alt {
+}
```
Comment (codewithrishu) [inline Components/alerts/alert.css]: not needed?

File: Components/alerts/alert.css [hunk 3]
```diff
@@ -0,0 +1,416 @@
...
+} */
+
+/* basic alert */
+
+.basic-alerts {
+	display: flex;
+	flex-direction: column;
+	/* justify-content: center; */
+	/* align-items: center; */
+	/* justify-items: center; */
```
Comment (ivinayakg) [inline Components/alerts/alert.css]: remove comments

File: Components/alerts/alert.css [hunk 4]
```diff
@@ -0,0 +1,416 @@
...
+} */
+
+.alert-error-title {
+	display: flex;
+	flex-direction: row;
+	position: relative;
+	background-color: rgb(56, 24, 24);
+	border-radius: 4px;
+	width: 50%;
+	padding: 0.6rem 0rem;
```
Comment (codewithrishu) [inline Components/alerts/alert.css:129]: no need to put unit after 0

File: Components/alerts/alert.css [hunk 5]
```diff
@@ -0,0 +1,416 @@
...
+	left: 1.1rem;
+}
+
+.p-error {
+	color: antiquewhite;
+	font-weight: 600;
+	margin-left: 2rem;
+}
+
+.p-warning {
```
Comment (ivinayakg) [inline Components/alerts/alert.css:101]: same as p.error

File: Components/alerts/alert.html [hunk 6]
```diff
@@ -0,0 +1,392 @@
...
+}, speed);
+});
+}
+removeFadeOut(document.querySelector(".close"), 2000);
+</script>
+
+
+<!-- script for codeblock -->
+<script type="text/javascript">
+(function () {
```
Comment (ivinayakg) [inline Components/alerts/alert.html:344]: put the js in a seperate file

File: Components/alerts/alert.html [hunk 7]
```diff
@@ -0,0 +1,392 @@
...
+src="../../images/svgs/error.svg"
+alt=""
+/>
+<p class="p-error-action">
+This is a error alert — <strong>check it out!</strong>
+</p>
+<input
+class="delete"
+type="image"
+src="../../images/svgs/delete.svg"
```
Comment (ivinayakg) [inline Components/alerts/alert.html:315]: use a formater for better code understanding

**Review by ivinayakg — APPROVED** [review]

File: Components/alerts/alert.html [hunk 8]
```diff
@@ -0,0 +1,392 @@
+<!DOCTYPE html>
```
Comment (codewithrishu) [inline Components/alerts/alert.html:1]: use beautify in this file

**Review by codewithrishu — APPROVED** [review]
in so many places unwanted code was present, remove that..other then that looks good to merge

**Pull request merged**